### ISSUE TITLE
Capacity to update a page with the new unique_id property

### ISSUE THREAD

**Issue by laHeud** [mariosimao/notion-sdk-php#256]
Hello, Notion recently introduced the ability to add ID properties in tables, similar to Jira. However, it causes a problem when we want to update a page because currently, when we update a page, all the properties are passed, resulting in an error. The solution I propose is to create a property type for 'unique_id' and enclose it in

```
 public function update(Page $page): Page
    {
        $updatableProps = array_filter($page->properties, function (PropertyInterface $p) {
            $notUpdatableProps = [ CreatedTime::class, LastEditedBy::class, LastEditedTime::class, CreatedBy::class, Unknown::class ];

            return (!in_array($p::class, $notUpdatableProps));
        });
```

As I tried with the Unknow::class.

Like that it works like a charm.

Wdyt about this solution ? 

<img width="441" alt="Capture d’écran 2023-06-20 à 18 52 53" src="https://github.com/mariosimao/notion-sdk-php/assets/2797509/1c390fb4-db56-4c1f-b206-44926922306a">

**Comment by laHeud** [comment]
I tested it locally, and it works well. I'm not sure if it's the right approach, but it seems good to me. I created a local branch where I added a property in Page and Database, and I added the UniqueId::class in the exclude properties of the update method.

**Comment by mariosimao** [comment]
Hi @laHeud!

I could not find any reference to Unique ID properties on the Notion API documentation, not even on the changelogs.

Could you open a PR with the new property that you already created? I will gladly add it to the SDK.

**Comment by laHeud** [comment]
Hi,

I would like to make a pull request, but I'm not sure about the procedure. Could you tell me which branch should I start from?

**Comment by mariosimao** [comment]
@laHeud, the procedure is simple:

1. Fork the repository
2. Start from branch `main`
3. Commit and push your changes
4. Open a pull request

[This article](https://opensource.com/article/19/7/create-pull-request-github) contains detailed information about creating pull requests.

**Comment by mariosimao** [comment]
Thanks for your feedback!

Fixed and released on v1.11.0
